src/containers/Dev/DevOptions.tsx

```diff
@@ -1,24 +1,13 @@
 import React, { CSSProperties } from 'react'
-import { Dropdown, Menu } from 'semantic-ui-react'
-import { useUserState } from '../../contexts/UserState'
-import useListTemplates from '../../utils/hooks/useListTemplates'
+import { Dropdown } from 'semantic-ui-react'
 import UserSelection from './UserSelection'
-import AppMenu from './AppMenu'
 
 const DevOptions: React.FC = () => {
-  const {
-    userState: { isLoading, templatePermissions },
-  } = useUserState()
-
-  const { filteredTemplates } = useListTemplates(templatePermissions, isLoading)
   return (
     <div id="dev-options" style={menuStyle}>
       <Dropdown item icon="user">
         <UserSelection />
       </Dropdown>
-      <Dropdown item icon="map">
-        <AppMenu templatePermissions={filteredTemplates} />
-      </Dropdown>
```

Comment:
Why did you remove this?

Reply:
i thought we would just click to dashboard now, should i reinstate it?

Reply:
Oh okay, so the Dashboard should show all the templates that are available in that list? If so, then that's cool, leave it out now.

Reply:
yep, it would match the list